presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -13,37 +13,34 @@
  */
```

Comment:
It would be nice to have https://github.com/prestosql/presto/issues/4510 implemented so we can have coverage as integration test.

Reply:
Yes, but this i'd still want to have product tests, as they allow me to cross-check results between Presto and Hive, especially because of subtleties like https://github.com/prestosql/presto/issues/5978